src/frontend/src/views/GroupMembersPage/components/MembersTab/index.tsx

```diff
@@ -54,9 +56,32 @@ const MembersTab = ({
     setTabValue(value);
   };
 
+  // TODO (mlila): api call
+  const invites = [
+    {
+      email: "erica@fake.com",
+      status: "pending",
+      dateSent: "2022-05-24",
+      role: "Member",
+    },
+    {
+      email: "frank@fake.com",
+      status: "expired",
+      dateSent: "2022-03-24",
+      role: "Member",
+    },
+  ];
```

Comment:
do we still need this stub?

Reply:
yep! I added the basic function to read invites, but it doesn't actually work yet (auth0 migration has not happened), so the last piece for me will be to double check that the function works as expected after migration and to take out these test values.